web-console/src/components/menu-checkbox/menu-checkbox.tsx

```diff
@@ -16,15 +16,26 @@
  * limitations under the License.
  */
 
-import { Checkbox, ICheckboxProps } from '@blueprintjs/core';
+import { MenuItem } from '@blueprintjs/core';
+import { IconNames } from '@blueprintjs/icons';
 import React from 'react';
 
-import './menu-checkbox.scss';
+export interface MenuCheckboxProps {
+  text: string;
+  checked: boolean;
+  onChange: () => void;
+}
+
+export function MenuCheckbox(props: MenuCheckboxProps) {
+  const { text, checked, onChange } = props;
 
-export function MenuCheckbox(props: ICheckboxProps) {
   return (
-    <li className="menu-checkbox">
-      <Checkbox {...props} />
-    </li>
+    <MenuItem
+      className="menu-checkbox"
+      icon={checked ? IconNames.TICK : IconNames.BLANK}
```

Comment:
If none of the menu items are checked, there's no visual clue that the menu item has a checkbox since it's blank. Maybe something like the `TICK CIRCLE` / `CIRCLE` icons could be used instead? Or, if this was changed to use the `Switch` component (like the other changes you made in the PR), would it have similar styling issues?

Reply:
I agree. This looks much better

![image](https://user-images.githubusercontent.com/177816/94728736-870e7400-0315-11eb-8187-47e8c03b6fe9.png)

Thanks for suggestion!